parquet-column/src/main/java/parquet/vector/RowBatch.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package parquet.vector;
+
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+import static parquet.Preconditions.checkNotNull;
+
+public class RowBatch {
```

Comment:
what is the order of columnVectors?
Shall we consider a:
ColumnVector getColumn(ColumnDescriptor descriptor)
or, just leave it as this is, assuming the order is exactly the same as the column appeared in requestedSchema?